### ISSUE TITLE
Doesn't work?

### ISSUE THREAD

**Issue by aditsu** [Rob--W/enablelegacy-firefox-addons#1]
I followed the steps and restarted firefox, but ChatZilla is still disabled.
I'm using firefox 60.2.2esr

**Comment by Rob--W** [comment]
The autoconfig hack has been patched (around Firefox 60). The tutorial on the master branch of this repository does not work any more and should not be used.

This tutorial was meant to help users to temporarily use legacy add-ons at the release of Firefox 57. With so many releases ahead of 56, the odds of running an unsupported legacy extension without problems are slim, even if you manage to re-enable legacy extensions.

**Comment by sergiomb2** [comment]
Hi @Rob--W , I think you may add this information on README.md , would be useful thanks , meanwhile I'm still using Firefox 56 and don't plan update it 
Thanks

**Comment by marius-klimantavicius** [comment]
I have just installed 60.3.0esr - I had to enable all three options from README to enable Tab mix plus, otherwise it was disabled (all legacy addons were disabled).

**Comment by sergiomb2** [comment]
So it works for you ?

**Comment by marius-klimantavicius** [comment]
Yes, instructions from master branch worked for me (60.3.0esr (64-bit) both Windows 10 and Windows 7) - my legacy addons were enabled.
- I added .cfg and -prefs.js
- Set `extensions.legacy.enabled` to `true` and `extensions.update.autoUpdateDefault` to `false`
- After restart my legacy add-ons were still disabled
- Set `xpinstall.signatures.required` to `false`
- After restart my legacy add-ons were enabled (I only care about Tab Mix Plus so I only made sure that it works).

I noticed that I needed `xpinstall.signatures.required` because on one pc where I simply modified all of the parameters it worked like a charm. But on the other I had only set `extensions.legacy.enabled` to `true` and `extensions.update.autoUpdateDefault` to `false` and it failed. I even tried modifying omni.ja from `patcher` branch to no avail. And only once I set `xpinstall.signatures.required` to `false` it worked like a charm.

**Comment by sergiomb2** [comment]
cool it working here also : 

```
cp enablelegacy.cfg  /usr/lib64/firefox/
cp enablelegacy-prefs.js /usr/lib64/firefox/defaults/preferences/ 
```
I used preferences directory instead pref directory just because will be easier to packaging . It works in both options. 

- about:config
- extensions.legacy.enabled to true
- xpinstall.signatures.required to false

After restart my legacy are working (saved passwords and tab mix plus and his sections manager ) 

Thanks.

**Comment by sergiomb2** [comment]
tab mix plus and his sections manager at the end doen't start up , but at least other things works

**Comment by sergiomb2** [comment]
saved-password-editor also have some problem feature of editing entry doesn't work . 
Skype extension also is not fully functional , I'm writing just in case someone want try it
